lib/kwtool_web/templates/home/index.html.eex

```diff
@@ -1,3 +1,25 @@
-<h2>Dashboard</h2>
-<p>You can only see this page if you are logged in</p>
-<p>You're logged in as <%= @conn.assigns.current_user.full_name %></p>
+<div class="container">
+  <div class="row">
+    <nav class="sidebar">
```

Comment:
Not sure about your design. If you plan to have a sidebar on other pages then it better to move `sidebar` and `content` elements to layout instead of template. Also, the scss should be in the layout directory too.

But if this only on this page then it's looking good to me 👍

Reply:
@hanam1ni !

Cool suggestion! I refactored the layout:
- Landing: for homepage
- Auth: for authentication
- App: for the main app of logged in user (including sidebar)